rotkehlchen/chain/ethereum/modules/zerox/constants.py

```diff
@@ -8,3 +8,16 @@
 ZEROX_FLASH_WALLET: Final = string_to_evm_address(
     '0x22F9dCF4647084d6C31b2765F6910cd85C178C18',
 )
+
+# The settler contract has different deployments and
+# the contract emits an event during deployment with the git commit of
+# the deployment contract version.
+# You can find this event in the deployment transaction:
+# https://etherscan.io/tx/0xfafcc9be3a8c6ceda26df29046b8651fa1298537b55c9b09fc5fd0d815f943e6#eventlog#236  # noqa: E501
+# https://github.com/0xProject/0x-settler/blob/b1dd3e25b305fe0b4ccda70ed7d85b797501a3f7/src/SettlerBase.sol#L69  # noqa: E501
+# The first git commit tested: 0xa6f39ee20f0c4dfe1265f5d203dfc4f3f05ca003000000000000000000000000
+# last commit tested: 0xffc129424fbe525c124e52cff5225afbfb610534000000000000000000000000
+SETTLER_ROUTERS: Final = [
+    string_to_evm_address('0x70bf6634eE8Cb27D04478f184b9b8BB13E5f4710'),
+    string_to_evm_address('0x0d0E364aa7852291883C162B22D6D81f6355428F'),
+]
```

Comment:
```suggestion
SETTLER_ROUTERS: Final = [
    string_to_evm_address('0x70bf6634eE8Cb27D04478f184b9b8BB13E5f4710'),  # commit 0xa6f39ee20f0c4dfe1265f5d203dfc4f3f05ca003000000000000000000000000
    string_to_evm_address('0x0d0E364aa7852291883C162B22D6D81f6355428F'),  # commit 0xffc129424fbe525c124e52cff5225afbfb610534000000000000000000000000
]
```
I think would make more sense to simply label the address with which commit it corresponds to like this.